Userland/Libraries/LibWeb/DOM/Element.cpp

```diff
@@ -2902,4 +2903,43 @@ void Element::inherit_counters()
     m_counters_set = move(element_counters);
 }
 
+// https://html.spec.whatwg.org/multipage/dom.html#the-lang-and-xml:lang-attributes
+StringView Element::lang() const
```

Comment:
Returning a `StringView` here is guaranteed to lead to a use-after-free. `StringView` is just a pointer an a length, it does not keep the string it points to alive. So when you return the result of e.g. `get_attribute` below, the `String` acquired from `get_attribute` will be freed immediately upon returning from this function.